src/fuzzing/mod.rs

```diff
@@ -0,0 +1,17 @@
+use num_bigint::ToBigInt;
+use num_traits::ToPrimitive;
+use rand::prelude::*;
+use rand_chacha::ChaCha8Rng;
+
+use crate::compiler::sexp::random_atom_name;
+use crate::util::number_from_u8;
+
+pub mod fuzzrng;
+
+pub fn make_random_u64_seed() -> u64 {
+    let mut rng = ChaCha8Rng::from_entropy();
```

Comment:
it looks suspicious to use actual entropy in a fuzzer or fuzzer utility. you really want fuzzers to be deterministic and derive all entropy from the input

Reply:
i will comment this that it isn't in the path of the fuzzer.  it's a way to get a random seed in case i need one to exercise the rest of the random generation code.  I used the slice from the fuzzer as the actual source of entropy in the fuzzer but it isn't available outside the fuzzer where i may want to meta-test this code.

i will put a warning on it ensuring the reader that it isn't in the fuzz path.

Reply:
*assure

Reply:
ok, great!